packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -227,12 +205,10 @@ async function updateDatasetContent() {
 	await fetchDataset();
 }
 
-async function updateDatasetName() {
-	if (transientDataset.value && !isEmpty(newName.value)) {
-		transientDataset.value.name = newName.value;
-		await updateDatasetContent();
-	}
-	isRenaming.value = false;
+function updateDatasetName(name: string) {
+	if (!transientDataset.value || isEmpty(name)) return;
```

Comment:
If tera-asset check if the new name is empty, we don't need to do it here.

```suggestion
	if (!transientDataset.value) return;
```